pages/workspaces/[workspaceId]/repository-insights/new.tsx

```diff
@@ -0,0 +1,25 @@
+import { useRouter } from "next/router";
+import InsightPage from "components/organisms/InsightPage/InsightPage";
+
+import useSession from "lib/hooks/useSession";
+import { WorkspaceLayout } from "components/Workspaces/WorkspaceLayout";
+
+const NewInsightPage = () => {
+  useSession(true);
+  const router = useRouter();
+  const workspaceId = router.query.workspaceId as string;
+
+  return (
+    <WorkspaceLayout workspaceId={workspaceId}>
+      <InsightPage workspaceId={workspaceId} />
```

Comment:
Not sure where the state of all the moving pieces are, but when I went to create an insight for a different workspace (not my personal workspace), it created the insight page in my personal one (confirmed by looking at the database tables) and not the workspace I had selected.

Maybe the insight hook needs to be updated to use `POST /v2/workspaces/{id}/insights`? Or that comes after this PR?

Reply:
Good catch. I will add that to this PR for the creation to use the other endpoint

Reply:
Coming in late, tried this out on `beta.app.opensauced.pizza` and this works as expected now!! Thanks Brandon!